### DISCUSSION TITLE
Relationships disappearing / rows deleting !

### DISCUSSION THREAD

**Discussion by R2bEEaton** [Budibase/budibase#7480] [Help]
I have some relationships that started disappearing after running an automation. From what I can tell, it happens after one of my automations triggers.

A row is updated in the Orders table.
The "Trip Check Complete" automation is triggered by this update, which does some logic then updates the Trips table.
Below is a screenshot of the only Update Row in that automation.

![image](https://user-images.githubusercontent.com/34921506/186974048-6435f23b-c1b0-4983-bbba-bb562a72bf18.png)

Orders <-> Trips are related many to one. The Trips column stores Orders in the Orders column. I know the Update Row is working because TripStatus changes to "Complete" correctly. The TripDriver relationship also disappears. But what's more - the actual Orders that the Trip were related to are deleted somehow. This doesn't seem possible to me because I don't have any Delete Row in any of my automation. I have been developing this app iteratively for two months, so I know my way around Budibase and I seriously cannot figure this out. I was messing with some things and I don't know why this suddenly stopped working.

I tried it on v1.2.47 and v1.2.52. Kind of panicking here, I really don't know what I did to cause this and I have made so many changes since my backup this morning.

**Comment by shogunpurple** [comment]
Hi @R2bEEaton,

it sounds to me like the update row automation could possibly be deleting the “other” relationship field from that row, causing the issues. 

If possible could you send an export of your app to support@budibase.com to aid reproduction, and we will try and get this one sorted ASAP.

Thank you!

**Reply by R2bEEaton** [reply]
Thanks for the response, I'm sending the email now. I've since reverted to my backup and recreated the changes I had made and now it's not deleting any rows or removing relationships. No idea what I did differently. I still think it would be nice to know what was causing the removal here if you'd like to give it a look over.

In my email will be a better explanation of what to do to recreate my experience. I've tried to remove the external Database passwords, which hopefully won't break much.
